controls/imageeditor/loading-image.md

```diff
@@ -1,53 +1,53 @@
 ---
 title: Display Images
 page_title: .NET MAUI ImageEditor Documentation - Display Images
-description: "Review the options to display images in the .NET MAUI ImageEditor control."
+description: Learn what are the options to display images in the Telerik UI for .NET MAUI ImageEditor control.
 position: 2
 slug: imageeditor-loading-image
 ---
 
-# Display Images in .NET MAUI ImageEdiotor
+# Display Images in .NET MAUI ImageEditor
 
 ImageEditor control enables you to visualize images using the following property:
 
 * `Source`(of type `Microsoft.Maui.Controls.ImageSource`): Specifies the source of the image. For more details about the Source property check the [Images in .NET MAUI](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/image?view=net-maui-7.0) article.
 
-The images could be loaded from:
+The images can be loaded from:
 
 * `File`
 * `Uri`
 * `Stream`
 * `Resources`
 
-**Load images for File**
+**Load Images for File**
```

Comment:
Either use a complete sentence or a caption to introduce the snippet. Don't use bold.